redbot/cogs/streams/streamtypes.py

```diff
@@ -162,7 +209,7 @@ def __init__(self, **kwargs):
             self.id = await self.fetch_id()
 
         url = TWITCH_STREAMS_ENDPOINT + self.id
-        header = {"Client-ID": str(self._token), "Accept": "application/vnd.twitchtv.v5+json"}
```

Comment:
From https://dev.twitch.tv/docs/v5#requests


For client IDs created on or after May 31, 2019, the only available version of the Kraken API is v5.  For client IDs created prior to May 31, 2019, use the `application/vnd.twitchtv.v5+json` header on your requests to access v5 of the Kraken API.

Probably shouldn't remove that header

Reply:
This is blocking for #3487 which will remove this issue entirely by moving to the helix API.